src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/integrationtest/AbstractDocumentTest.java

```diff
@@ -38,7 +37,7 @@
 import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
 import static org.springframework.http.HttpHeaders.AUTHORIZATION;
 import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
-import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
```

Comment:
APPLICATION_JSON_UTF8_VALUE is deprecated and replaced with APPLICATION_JSON_VALUE